safe_transaction_service/history/tests/test_internal_tx_indexer.py

```diff
@@ -53,6 +55,10 @@ def test_internal_tx_indexer_provider(self):
                 (InternalTxIndexer, InternalTxIndexerWithTraceBlock),
             )
 
+    def return_sorted_blocks(self, hashes: HexStr):
+        block_dict = {block["hash"].hex(): block for block in block_result}
+        return [block_dict[provided_hash] for provided_hash in hashes]
```

Comment:
Not sure how to read this :D but just wondering if there is nothing here that is "lazily" evaluated that could return something unexpected

Reply:
This is to return the mock. Previously it was easy, as mock returned the blocks sorted by `block_number`. Now `hashes` are used, so sorting of the blocks can vary